frontend/packages/pipelines-plugin/src/components/shared/workspaces/__tests__/WorkspaceResourceLinkList.spec.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { ShallowWrapper, shallow } from 'enzyme';
+import { ResourceLink } from '@console/internal/components/utils';
+import { SecretModel, PersistentVolumeClaimModel, ConfigMapModel } from '@console/internal/models';
+import { taskRunWithWorkspaces } from '../../../taskruns/__tests__/taskrun-test-data';
+import WorkspaceResourceLinkList, {
+  WorkspaceResourceLinkListProps,
+} from '../WorkspaceResourceLinkList';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+describe('WorkspaceResourceLinkList', () => {
+  let workspaceResourceListWrapper: ShallowWrapper<WorkspaceResourceLinkListProps>;
+  let workspaceResourceListWrapperProps: WorkspaceResourceLinkListProps;
+
+  it('Should not render dd or dt heading if no workspaces are found', () => {
```

Comment:
Good use case for not explaining the expected output, speak to the reason for the test.

`it('should not render if there are no workspaces'`